### DISCUSSION TITLE
Universal Rofi Flags

### DISCUSSION THREAD

**Discussion by ghost** [davatorium/rofi#1609] [Q&A] [answered]
I have the following in `.Xresources`:
```
Xft.dpi: 120
*dpi: 120
```
Which I am aware is deprecated in favor of the `-dpi X` flag. I use `-dpi 1` on normal invocation of `rofi`, but I can't set a dpi flag on some miscellaneous rofi extension, e.g. `rofi-pass`. I'm sure there's ways of manually doing this for each extension, but is there a way to set a universal flag to all `rofi` invocations?

**Comment by DaveDavenport** [comment, marked as answer]
From the readme: https://github.com/davatorium/rofi#configuration
The manpage: https://github.com/davatorium/rofi/blob/next/doc/rofi.1.markdown#configuration

**Reply by ghost** [reply]
The problem is there is no option described in the docs/manpages where the dpi can be set in `config.rasi`. Changing the command line options depends on the extension. 

I understand that the `config.rasi` format has had plenty of breaking changes in the past 2 years, but I am looking at current manpages.

**Reply by DaveDavenport** [reply]
Have you tried running: 

```bash
rofi -dump-config
```
as suggested everywhere. 

It generates a template config file that even contains dpi setting (commented out).

If you run:
```bash
rofi -dpi 128 -dump-config 
```
it will give you the config file template with the dpi value uncommented and set to 128.

**Reply by ghost** [reply]
Thanks!

**Reply by DaveDavenport** [reply]
I tried to make it easy.
